Makefile

```diff
@@ -238,20 +242,6 @@ $(OUTPUT_DIR)/extr: extr.c $(MUPDF_LIB) $(MUPDF_DIR)/include $(JPEG_LIB) $(FREET
 		$(CFLAGS) -Wl,-rpath,'libs' -o $@ extr.c \
 		-lmupdf $(JPEG_LIB_LINK_FLAG) $(FREETYPE_LIB_LINK_FLAG)
 
-# ===========================================================================
-# helper target for creating standalone android toolchain from NDK
-# NDK variable should be set in your environment and it should point to
-# the root directory of the NDK
-#
-# --deprecated-headers is necessary in NDK 15, but will fail in 12-14
-android-toolchain:
-ifneq ($(wildcard $(NDK)/build/tools),)
-	$(NDK)/build/tools/make_standalone_toolchain.py --force --install-dir=$(ANDROID_TOOLCHAIN) \
-		--arch $(ANDROID_ARCH) --api $(NDKABI) --deprecated-headers || \
-	$(NDK)/build/tools/make_standalone_toolchain.py --force --install-dir=$(ANDROID_TOOLCHAIN) \
-		--arch $(ANDROID_ARCH) --api $(NDKABI)
-endif
-
```

Comment:
Please don't remove this.

Reply:
Newer NDKs don't support this: there's no need for it, as the toolchains can directly be used.

Reply:
Starting with NDK r19c: see https://github.com/android/ndk/wiki/Changelog-r19.

Reply:
In that case don't forget to remove it from .PHONY at the bottom.

Reply:
Done.